molecule.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <iostream>
+#include <iomanip>
+#include <vector>
+#include <algorithm>
+#include "atom.h"
+
+using namespace std;
+
+class Molecul {
+public:
+    string name;
+    vector <Atom> atoms;
+    Molecul(string name)
+    {
+        this->name = name;
+    }
+    void Add_atom(Atom& a) {
+        atoms.push_back(a);
+    }
+    void sort_by_atomic_mass();
+    double findAverageMass();
+};
+
+void Molecul::sort_by_atomic_mass() {
+    sort(atoms.begin(), atoms.end(), [](Atom& a, Atom& b) { return a.atomic_mass < b.atomic_mass; });
+    cout << endl << "sorted Atoms:" << endl;
+    for (auto i : atoms) {
+        cout << i.name << " = " << i.atomic_mass << setw(7);
+    }
+}
+double Molecul::findAverageMass() {
```

Comment:
I see two different conventions for function naming:
double Molecul::findAverageMass() {
 void Molecul::sort_by_atomic_mass() {